.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.5.9
+ruby 2.7.1
```

Comment:
Thanks for the change! Mind putting this in another PR? This seems like something someone may want to track down later and I don't want it to get lost in this one.

Reply:
Actually, take that back — since [we still support Ruby 2.5 as the minimum tested version](https://github.com/mcmire/super_diff/blob/master/.github/workflows/super_diff.yml#L21), we probably want to leave this alone :)

Reply:
I reverted it in [2450859](https://github.com/mcmire/super_diff/pull/146/commits/2450859b8147b3b70ce755e03e3650cd4555422f)

Reply:
I will fix this error in the test suite

```
            NoMethodError:
              undefined method `try' for #<Hash>
              Did you mean?  pry
```

Reply:
The remainder that are failing are probably due to an issue with ruby version + minitest version issues, I pushed up https://github.com/mcmire/super_diff/pull/148 to try and fix the issue